src/libsync/owncloudpropagator.cpp

```diff
@@ -422,9 +422,9 @@ void OwncloudPropagator::start(SyncFileItemSet &&items)
                 continue;
             } else if (item->_instruction == CSYNC_INSTRUCTION_IGNORE) {
                 continue;
-            } else if (item->_instruction == CSYNC_INSTRUCTION_RENAME) {
+            } else if (item->_instruction != CSYNC_INSTRUCTION_RENAME) {
                 // all is good, the rename will be executed before the directory deletion
```

Comment:
this part is really fishy, first I thought it was missing a continue but I think the condition was just to prevent the warning